cli/src/cmd/forge/verify/etherscan/mod.rs

```diff
@@ -405,7 +434,7 @@ impl EtherscanVerificationProvider {
                 &read_constructor_args_file(constructor_args_path.to_path_buf())?,
             )?
             .to_hex::<String>();
-            return Ok(Some(encoded_args[8..].into()))
+            return Ok(Some(encoded_args[8..].into()));
```

Comment:
I think you've got `Format on save` turned on using the normal `cargo fmt` command. My recommendation would be:
- Turn off format on save for rust
- Instead for foundry run `cargo +nightly fmt`

Reply:
Thank you for pointing it out. I'm working on it 😄